### ISSUE TITLE
Build fails on Circle

### ISSUE THREAD

**Issue by CJ-Wright** [conda-forge/doct-feedstock#4]
Maybe get a re-render?

**Comment by CJ-Wright** [comment]
@ericdill thoughts?
https://circleci.com/gh/conda-forge/doct-feedstock/11

**Comment by ericdill** [comment]
@CJ-Wright no idea. Can you finish #2? That should fix any issues

**Comment by CJ-Wright** [comment]
Is #2 still relevant? It has a 2.1.0 rerender, but the most recent pull has a 2.3.1 rerender. I can rerender with 2.3.2 if you want. But I think we should close #2

**Comment by ericdill** [comment]
Yes. please rerender with 2.3.2. It looks like there's also a version bump necessary on #1. Can you do that as well?

**Comment by ericdill** [comment]
Go ahead and close both PRs. Or if you're not a member of this repo then add yourself to the recipe-maintainers

**Comment by CJ-Wright** [comment]
I think I am listed already, but thank you!
